src/main/java/frc/team2412/robot/Robot.java

```diff
@@ -145,7 +147,11 @@ public void autonomousInit() {
 
 		// Checks if FMS is attatched and enables joystick warning if true
 		DriverStation.silenceJoystickConnectionWarning(!DriverStation.isFMSAttached());
-		autoChooser.getSelected().schedule();
+		// autoChooser.getSelected().schedule();
+		System.out.println(AutoLogic.getSelected() != null);
```

Comment:
Ideally, this log message would be cleanup or be more descriptive.

Reply:
I've removed it because that was just originally used for debug purposes